resources/ansible/uploaders.yml

```diff
@@ -6,4 +6,24 @@
   roles:
     - role: safe_user
       become: True
-    - uploaders
\ No newline at end of file
+    - role: uploader-metrics
+      become: True
+    - uploaders
+
+  tasks:
+    # Something is wrong with the journal service on Ubuntu that causes no
+    # output to be produced when running `journalctl`. The solution seemed
+    # to be restarting the service.
+    - name: restart systemd-journald
+      become: True
+      ansible.builtin.systemd:
+        name: systemd-journald
+        state: restarted
+        enabled: yes
+    # The Telegraf service seems to need to be rebooted for metrics to start transmitting.
+    - name: restart telegraf
+      become: True
+      ansible.builtin.systemd:
+        name: telegraf
+        state: restarted
+        enabled: yes
```

Comment:
This is restarted on uploaders.yml but not part of nodes.yml.  Should this be in both or the issue is specific to uploader droplets only?

Reply:
Yeah, this is also in `nodes.yml`. It's down at the bottom of the file.

Reply:
Aaah yes, I thought I had expanded the full file of `nodes.yml` but I didn't. All good.